starknet_devnet/starknet_wrapper.py

```diff
@@ -640,13 +697,13 @@ async def update_pending_block(self, state_update: BlockStateUpdate = None):
             state_update=state_update,
         )
 
-    async def generate_latest_block(self) -> StarknetBlock:
+    async def generate_latest_block(self, block_hash=None) -> StarknetBlock:
```

Comment:
I don't really like this approach with adding a new argument, but if we're going to use it, it could be mentioned in the docstring.

Reply:
Added doc here a1f6b32a7e4b36514cd35c509e3ab91872cfc273